lib/fog/libvirt/models/compute/network.rb

```diff
@@ -27,6 +27,17 @@ def save
         def shutdown
           service.destroy_network(uuid)
         end
+
+        def to_xml
```

Comment:
Further testing told me `save` isn't even implemented now so I'm not too worried about this.